### DISCUSSION TITLE
55/5000  My GitHub Copilot has been unable to cancel the use of this interface after clicking on the trial, but it shows in the bill that I have the student qualification to use it for free

### DISCUSSION THREAD

**Discussion by Users00001** [community/community#114351] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I was trying out GitHub Copilot for a month, during which I did student certification, but now that this month has expired, GitHub Copilot tells me I can't use it anymore, and clicking the Open button won't redirect me to the open page. In addition, my GitHub Copilot has been unable to cancel the use of this interface after clicking on the trial, as shown in the picture below, but it shows in the bill that I am eligible to use it for free. How can I solve this problem?
![36c38d6b4db03c3df6b4f88dcf506be](https://github.com/community/community/assets/99785971/4429d418-972f-4355-a9d7-7fbfc47b4f5c)
![58e0e8b27218a2fe5f4e8de360385c6](https://github.com/community/community/assets/99785971/44667d2b-20b9-4439-ad63-0398ec645de3)

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
